generic/tohil.c

```diff
@@ -977,7 +1047,13 @@ TohilExecEvalPython(int startSymbol, Tcl_Interp *interp, int objc, Tcl_Obj *cons
         return tohil_tcl_return(interp, TCL_ERROR);
     }
     Tcl_DString ds;
-    const char *cmd = tohil_TclObjToUTF8DString(interp, objv[1], &ds);
+    char *cmd = tohil_TclObjToUTF8DString(interp, objv[1], &ds);
+    if (startSymbol == Py_file_input) {
```

Comment:
I was going to add a flag indicating "don't undent" and pass it in from TohilEval_Cmd but the code already uses startSymbol semantically in error messages so this is consistent with the existing code.